velox/connectors/hive/HiveDataSink.h

```diff
@@ -207,7 +208,7 @@ class HiveDataSink : public DataSink {
   const CommitStrategy commitStrategy_;
   // Parameters used by writers, and thus are tracked in the same order
   // as the writers_ vector
-  std::vector<std::shared_ptr<HiveWriterInfo>> writerInfo_;
+  std::vector<std::shared_ptr<HiveWriterInfo>> writerInfos_;
```

Comment:
Plural of information?

Reply:
It is ok to name like this in variable naming such as groupingKeyInfos_.